cloud_functions/climateiq_data_export_cf/main_test.py

```diff
@@ -74,32 +76,41 @@ def test_export_model_predictions_invalid_study_area(
     }
     event = CloudEvent(attributes, data)
 
-    mock_study_area = mock.Mock()
-    mock_study_area.exists = True
-    expected_metadata = {
+    # Build mock Storage object
+    predictions = '{"instance": [1], "prediction": [[1, 2, 3], [4, 5, 6]]}\n'
+    with mock_storage_client().bucket("").blob("").open() as mock_fd:
+        mock_fd.__iter__.return_value = iter(predictions.splitlines())
+
+    # Build mock Firestore document
+    metadata = {
         "name": "study_area_name",
         "crs": "EPSG:32618",
-        "chunks": {},  # Missing "cell_size" required field
-    }
-    mock_study_area.to_dict.return_value = expected_metadata
-    mock_firestore_client.return_value.collection.return_value.document \
-        .return_value.get.return_value = (
-            mock_study_area
-        )
+        "chunks": {
+            "chunk-id": {
+                "row_count": 2,
+                "col_count": 3,
+                "x_ll_corner": 500,
+                "y_ll_corner": 100,
+            }
+        },
+    }  # Missing "cell_size" required field
+    mock_firestore_client().collection().document().get.return_value = (
+        MagicMock(to_dict=lambda: metadata)
+    )
```

Comment:
Looks so much nicer with the mocks cleaned up! This looks like it could still be simplified to
```python
mock_firestore_client().collection().document().get().to_dict.return_value = metadata
```
?
same below in several spots